src/applications/personalization/profile-2/sass/profile-info-table.scss

```diff
@@ -14,18 +14,20 @@ $corner-size: 4px;
     border: 1px solid;
     border-top: 0;
 
-    h4 {
+    dt {
       @include media($medium-screen) {
         width: 208px;
         flex-shrink: 0;
       }
     }
   }
 
-  div.table-row:first-child {
-    border: 1px solid;
-    border-top-left-radius: $corner-size;
-    border-top-right-radius: $corner-size;
+  dl:first-child {
```

Comment:
After changing the markup of this component, it took me a little too long to realize how to keep the existing styling where the top row of the table has rounded corners if the component didn't render a title. But this simple change makes it work!